src/app/workflow/workflow-file-editor/workflow-file-editor.component.ts

```diff
@@ -142,7 +142,10 @@ export class WorkflowFileEditorComponent extends FileEditing {
       },
       (error: HttpErrorResponse) => {
         if (error.status === 413) {
-          this.alertService.detailedError(error, 'Cannot save new version: versions have a 60 kilobyte limit');
+          this.alertService.customDetailedError(
```

Comment:
A comment here would be nice, mentioning that nginx returns HTML for this error. I know it's in  your commit message (good!), but if somebody is looking at this one year from now, they'll wonder why this error is handled differently than all the others in the codebase, and it would be nice if they could do so without having go through the Git history.

Reply:
Done.